resources/views/add_car.blade.php

```diff
@@ -48,13 +48,16 @@
           <div class="form-group mb-3 row">
             <label for="" class="form-label col-md-2 fw-bold text-md-end">Category:</label>
             <div class="col-md-10">
-              <select name="categoryName" id="" class="form-control">
+              <select name="category_id" id="" class="form-control">
                 <option value="">Select Category</option>
                 @foreach($categories as $category)
-                <option value="{{$category->id}}">{{$category->categoryName}}</option>
+                <option value="{{$category->id}}" {{ old('category_id') == $category->id ? 'selected' : '' }}>{{$category->categoryName}}</option> 
+                <!-- another way as the edit blade: -->
+                <!-- <option value="{{$category->id}}" @selected(old('category_id') == $category->id)>{{$category->categoryName}}</option> -->
+                
                 @endforeach
               </select>
-              @error('category')
+              @error('category_id')
```

Comment:
the first one to notice it 
![images](https://github.com/user-attachments/assets/261bf3b7-2c27-4370-8a60-55859b3e406f)